.github/workflows/countline.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""
+This is an countline script.
+
+It runs on lib and test directory to find files above given lines of code
+and print all files above given lines of code.
+"""
+
+import os
+import sys
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
Put all the `argparse` related lines in a separate function. Avoid global variables in your scripts.

Reply:
Done